sos/plugins/__init__.py

```diff
@@ -860,6 +860,26 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         self._log_debug("collecting journal: %s" % journal_cmd)
         self._add_cmd_output(journal_cmd, None, None, timeout)
 
+    def add_udev_info(self, device, attrs=False):
```

Comment:
This is a big new interface in the `Plugin` class: it deserves its own commit, and description.

Reply:
It might be good to also support `-p` so that we can directly replace existing `add_cmd_output()` calls for `udevadm` with this method. This also needs some thought as to how it's going to function in a container - is `/sys` mounted where we'd expect?